Specifications/Applications/for_Feature/when_creating.cs

```diff
@@ -16,7 +16,7 @@ public class when_creating
             feature = new Feature(module_mock.Object, name);
         };
 
-        It should_set_the_name = () => ((string) feature.Name).ShouldEqual(name);
+        It should_set_the_name = () => ((string) feature.Name).ShouldEqual(name.Value);
```

Comment:
Not really part of this pull request, but I don't like the model of passing in the module to the Feature and having it add itself to the Module.  Double dispatch is ok, but not in a constructor.